common/lib/capa/capa/tests/test_responsetypes.py

```diff
@@ -2173,7 +2175,6 @@ def test_valid_xml(self):
         Test that `build_problem` builds valid xml
```

Comment:
Does this test even actually test that? How can it be a test without any sort of assertion?